sdk/synapse/Azure.Analytics.Synapse.Spark/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ### Breaking Changes
 
 ### Bugs Fixed
+- Enhance Long Running Operation (LRO) logic for `SparkBatchClient` to support both scenarios of job submission and job execution.
```

Comment:
I would add this to the "Features Added" section instead of the "Bugs Fixed" section, since it provides a new way of starting the CreateSparkBatchJob operation.

Reply:
Agree. This change is more like a new feature, though it does fix a bug in the PR: https://github.com/Azure/azure-sdk-for-net/pull/17920. In that PR, a new state "starting" is added as end state. This is neither correct for job submission scenario (in this case, running should also be an end state) nor correct for job execution scenario (in this case, starting is not an end state).

Reply:
Fixed